internal/validate/row/hasher.go

```diff
@@ -8,10 +8,10 @@ import (
 	"github.com/CDCgov/data-exchange-csv/cmd/internal/constants"
 )
 
-func ComputeHash(row []string, delimiter string) string {
+func ComputeHash(row []string, delimiter rune) string {
 	separator := constants.COMMA
 
-	if delimiter == string(constants.TAB) {
```

Comment:
Like the refactoring to using rune data types. I'm seeing casting stuff to strings when they should be runes to start with

Reply:
Its necessary here to convert a `rune` to a `string` because `strings.Join()` expects slice of strings.